tests/app/main/views/test_send.py

```diff
@@ -2615,6 +2588,40 @@ def test_check_notification_shows_preview(
     # assert tour not visible
     assert not page.select(".banner-tour")
 
+    # post to send_notification with help=0 to ensure no back link is then shown
+    assert page.form.attrs["action"] == url_for(
+        "main.preview_notification",
+        service_id=service_one["id"],
+        template_id=fake_uuid,
+    )
+
+    assert normalize_spaces(page.select_one("main [type=submit]").text) == ("Preview")
+
+
+@pytest.mark.parametrize("when", ["", "2016-08-25T13:04:21.767198"])
+def test_preview_notification_shows_preview(
+    client_request,
+    service_one,
+    fake_uuid,
+    mock_get_service_template,
+    when,
+):
+    with client_request.session_transaction() as session:
+        session["recipient"] = "2028675301"
```

Comment:
If you can replace these probably real phone numbers with 555-555-5555 and it doesn't break the test, might be a good idea.  I know some tests loop through phone number validation and will fail with 555, but we should work towards cleaning these numbers up when we can.